readme.md

```diff
@@ -73,3 +73,55 @@ bot.once('spawn', () => {
 * Dynamic harvest/mining paths
 * Sprint
 * Actual move speed per block
+
+## API
+Considering there are a lot of deep changes that are being worked on, it could take some time before it's done
+
+Also, **for now**, there is only the `pathfinder` module, `movements` and `goals` still need to be done
+
+
+# Functions:
+
+### bot.pathfinder.bestHarvestTool(block)
+Returns the best harvest tool in the inventory for the specified block
+ * `Return` - ?
+ * `block` - Block instance
+
+### bot.pathfinder.getPathTo(movements, goal, done, timeout)
+Returns a Path instance
+ * `Return` - Path instance
+ * `movments` - Movements instance
+ * `goal` - Goal instance
```

Comment:
while true, perhaps there should be a section on making a `Goal`, also while on this topic there could even be a section about extending the `Goal` class yourself to make a custom goal